web-components/src/components/tabs/IconTabs.tsx

```diff
@@ -67,23 +79,26 @@ const IconTabs: React.FC<IconTabsProps> = ({
               icon={tab?.icon}
               hidden={tab?.hidden}
               size={size}
+              href={tab.href}
+              linkComponent={linkComponent}
               sxInner={{ ...sxs?.tab?.inner }}
               {...a11yProps(index)}
             />
           ))}
         </StyledTabs>
       </Box>
-      {tabs.map((tab, index) => (
-        <TabPanel
-          key={index}
-          value={value}
-          index={index}
-          hidden={tab.hidden || value !== index}
-          sxs={{ ...sxs?.panel }}
-        >
-          {tab.content}
-        </TabPanel>
-      ))}
+      {hasContent &&
```

Comment:
Allow IconTabs to be used with tabs only for usage with react-router and nested routes.